packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -503,6 +504,15 @@ class RenderParagraph extends RenderBox with ContainerRenderObjectMixin<RenderBo
     markNeedsLayout();
   }
 
+  /// TextOverflow.ellipsis's value.
```

Comment:
Maybe: The string to replace the omitted text, when `textOverflow` is set to `TextOverflow....` and the text overflows.

Reply:
also consider describing the default value, as well as edge cases such as what happens if the ellipsis string itself overflows.